javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -193,14 +203,19 @@ module NeighborhoodBodies {
   /**
    * Gets the body token feature limited to the part of the function body that lies under `rootNode` in the AST.
    *
-   * This is a string of space-separated natural language tokens (AST leaves) in the order that they appear in the source code for the AST subtree rooted at `rootNode`. This is equivalent to the portion of the code that falls under
- * the AST subtree rooted at the given node, except that non-leaf nodes (such as operators) are excluded.
+   * This is a string of space-separated natural language tokens (AST leaves) in the order that they
+   * appear in the source code for the AST subtree rooted at `rootNode`. This is equivalent to the
+   * portion of the code that falls under the AST subtree rooted at the given node, except that
+   * non-leaf nodes (such as operators) are excluded.
+   *
+   * If a function has more than `maxNumBodySubtokens` body subtokens, then featurize it as absent.
+   * This approximates the behavior of the classifer on non-generic body features where large body
+   * features are replaced by the absent token.
    */
   string getBodyTokenFeatureForNeighborhoodNode(DatabaseFeatures::AstNode rootNode) {
-    // If a function has more than 256 body subtokens, then featurize it as absent. This
-    // approximates the behavior of the classifer on non-generic body features where large body
-    // features are replaced by the absent token.
-    if count(DatabaseFeatures::AstNode node, string token | bodyTokens(rootNode, node, token)) > 256
+    if
```

Comment:
Do we actually need `getBodyTokenFeatureForNeighborhoodNode`, or can we reuse `getBodyTokenFeatureForEntity`?